contents/Model as a judge/Reward Model.md

```diff
@@ -0,0 +1,52 @@
+# Reward Model Based Evaluation
+
+## What is a Reward Model?
+
+Reward models are a type of model that outputs a reward score for a given prompt and completion.
+They are often used in the context of Reinforcement Learning (RL), where the goal is to maximize the reward signal.
+
+The most common type of Reward Model is the so-called Bradley-Terry model, which outputs a single score with the following interpretation:
+
+$$p(\text{completion b is better than completion a}) = \text{sigmoid}(\text{score}_b - \text{score}_a)$$
+
+This model has the advantage of requiring only pairwise comparisons, which are easier to collect.
+However, as these models are trained only on comparisons, it is important to keep in mind that scores across different prompts should not be compared,
+ as they can be arbitrarily offset.
+
+Some reward models such as [SteerLM](https://arxiv.org/abs/2311.09528) output one or more absolute scores, which can be used to evaluate the model directly without the need for paired completions.
+These models can be easier to use for evaluation, but are also harder to collect data for. 
+More recently models have been proposed that output both absolute and relative scores, such as [HelpSteer2-Preference](https://arxiv.org/abs/2410.01257) and [ArmoRM](https://arxiv.org/abs/2406.12845).
+
+Another type of reward model are pair-wise models, which output a single score for a pair of completions ([example](https://huggingface.co/RLHFlow/pair-preference-model-LLaMA3-8B)).
```

Comment:
Thanks for the reference!

> the average score from a very big reward model on a fixed set of eval prompts

This is the main thing I tried to describe, with the caveat

* Average (BT) score, easy but can be affected by outlier scores and differences in confidence inherent in prompts, since scores are in logit space. 
  *  -> use win rate to avoid this, a little granular
     *  -> use mean win probability, smooth and not so affected by outliers.
   
Of course average absolute score doesn't have outliers, but the models tend to be a little weaker.